metric/system/process/process.go

```diff
@@ -166,10 +167,13 @@ func (procStats *Stats) GetSelf() (ProcState, error) {
 // pidIter wraps a few lines of generic code that all OS-specific FetchPids() functions must call.
 // this also handles the process of adding to the maps/lists in order to limit the code duplication in all the OS implementations
 func (procStats *Stats) pidIter(pid int, procMap ProcsMap, proclist []ProcState) (ProcsMap, []ProcState) {
+	// That's the method that fails, however it seems to return a partial state
```

Comment:
This looks like maybe a leftover or debugging comment? If not could it be clarified